docs/en/ingest-management/elastic-agent/configuration/autodiscovery/kubernetes-hints-autodiscover.asciidoc

```diff
@@ -115,8 +115,7 @@ volumes:
 ...
 ----
 
-An init container is also required to download the hints templates.
-The init container is already defined, so uncomment the respective section:
+Uncomment the init container section to download the hints templates:
```

Comment:
Why is this change needed? It's too directive and not 100% accurate. The init container is responsible for "downloading" the templates but the way that it's phrased in this change is like "execute this command to download" which does not reflect the reality.

Reply:
Reverted